sdk/communication/azure-communication-networktraversal/README.md

```diff
@@ -0,0 +1,134 @@
+    .connectionString(connectionString)
+    .buildClient();
+```
+
+## Key concepts
+
+`CommunicationRelayClient` and `CommunicationRelayAsyncClient` provide the functionalities to manage users and user tokens.
+
+## Examples
+
+### Getting a new Relay Configuration
+Use the `createUser` function to create a new user from CommunicationIdentityClient
+Use the `getRelayConfiguration` function to get a Relay Configuration
+
+<!-- embedme ./src/samples/java/com/azure/communication/networktraversal/ReadmeSamples.java#L73-L74 -->
+```java
+CommunicationIdentityClient communicationIdentityClient = new CommunicationIdentityClientBuilder()
+            .connectionString(connectionString)
+            .buildClient();
+
+CommunicationUserIdentifier user = communicationIdentityClient.createUser();
+System.out.println("User id: " + user.getId());
+
+CommunicationRelayConfiguration config = communicationRelayClient.getRelayConfiguration(user);
+```
+
+## Troubleshooting
+
+All user token service operations will throw an exception on failure.
+<!-- embedme ./src/samples/java/com/azure/communication/networktraversal/ReadmeSamples.java#L139-L143 -->
+```java
+try {
+    CommunicationUserIdentifier user = communicationIdentityClient.createUser();
+    CommunicationRelayClient communicationRelayClient = createCommunicationNetworkTraversalClient();
+    CommunicationRelayConfiguration config = communicationRelayClient.getRelayConfiguration(user);
+} catch (CommunicationErrorResponseException ex) {
+    System.out.println(ex.getMessage());
+}
+```
+
+## Next steps
+
+Please take a look at the [samples][samples] directory for detailed examples of how to use this library to manage relay configuration
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a [Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights to use your contribution.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
+
+<!-- LINKS -->
+[cla]: https://cla.microsoft.com
+[coc]: https://opensource.microsoft.com/codeofconduct/
+[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
+[coc_contact]: mailto:opencode@microsoft.com
+[product_docs]: https://docs.microsoft.com/azure/communication-services/
+[api_documentation]: https://aka.ms/java-docs
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2Fsdk%2Fcommunication)
```

Comment:
```suggestion
![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2Fsdk%2Fcommunication%2Fazure-communication-networktraversal%2FREADME.png)
```

Reply:
I'll change it to that after checking in, for one of the other's (JS or Python) it checked that links were not broken, so in order to avoid any issue like that, I'll just leave it as it is for now.